ExampleMod/Content/Items/Accessories/ExampleShield.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace ExampleMod.Content.Items.Accessories
 {
-	[AutoloadEquip(EquipType.Shield)]
+	[AutoloadEquip(EquipType.Shield)] // load sprite as a shield when eqipped on the player
```

Comment:
Please use capitalization and periods consistently in your comments. There are also inconsistencies in spacing after double forward slashes in same files.